climpred/constants.py

```diff
@@ -26,6 +31,7 @@
     'acc': 'pearson_r',
     'pval': 'pearson_r_p_value',
     'pvalue': 'pearson_r_p_value',
```

Comment:
Should we add an alias here for the spearman pvalue?

Reply:
To should come up with a clever naming. Or we just leave it as is assuming Pearsonr to be the standard. I don’t mind.

Reply:
I think these should stay as is since Pearson R is the standard. Maybe also offer "s_pval" and "s_pvalue" or something similar for spearman and mention that in the docs as the keyword.

Reply:
Do you mind changing oval to p_pval?

Reply:
I think that would be fine to be explicit and consistent.

Reply:
Or maybe add `p_pval` and leave the `pval` so that we don't add a breaking change out of nowhere.

Reply:
I don’t like the breaking change but still that would be better than having this inconsist naming.

Reply:
Okay then do `p_pval` and `s_pval` and add a header to the CHANGELOG called "deprecated" and mention that `pval` no longer works. If you wanted to be really careful you could throw a DeprecationError if someone types `pval` to explain the change.